cpp/dolfinx/fem/Expression.h

```diff
@@ -82,22 +88,18 @@ class Expression
   }
 
   /// Evaluate the expression on cells
-  /// @param[in] cells Cells on which to evaluate the Expression
+  /// @param[in] active_cells Cells on which to evaluate the Expression
```

Comment:
`cells` is nicer than `active_cells`. Not sure what 'active' means in this context.